### DISCUSSION TITLE
Question regarding possibility to output versioning info to a file

### DISCUSSION THREAD

**Discussion by thomasbeste** [GitTools/GitVersion#3010] [Q&A]
Hello sirs,

I am using Azure pipelines, to build and deploy a software package using GitVersion.

I would like to write the version output based on YAML, the same way as the argument switch /output file would do.
Then i would be able to make an endpoint on my REST services, where i could feed this file to the requester, thus easily showing what version the software currently has

for now, this is my take - not sure it is correct, since i have a really hard time finding documentation on this:

      - task: gitversion/execute@0
        displayName: Determine Version
        inputs:
          output: file,json
          useConfigFile: true
          configFilePath: 'GitVersion.yml'

could anyone please give me some pointers in the correct direction?

Thanks in advance :-)

**Comment by asbjornu** [comment]
You can find the supported YAML properties in the [documentation for the AzDO Task](https://github.com/GitTools/actions/blob/main/docs/examples/azure/gitversion/execute/usage-examples.md). I don't think `output:` is supported, but I think something like `additionalArguments: -output file` should work.
